dev-requirements/build.txt

```diff
@@ -1,2 +1,2 @@
-setuptools~=65.1.0
+setuptools~=65.2.0
```

Comment:
It is beyond me how they have so many releases. Should be good enough to just require above a specific version.


```suggestion
setuptools>=65.2.0
```